### ISSUE TITLE
HttpMessageNotReadableException when attempting to POST /games/{id}

### ISSUE THREAD

**Issue by lausberger** [lausberger/rest-game#2]
```
POST /battles/xyz {"action": "ATTACK"}
```

Will result in 
```
Resolved [org.springframework.http.converter.HttpMessageNotReadableException: JSON parse error: Cannot deserialize value of type `com.lucas.restgame.model.BattleAction` from Object value (token `JsonToken.START_OBJECT`)]
```

Cause: Jackson is attempting to deserialize entire RequestBody object as a BattleAction.

Possible Solutions: An class specific to this method which contains a BattleAction, or deserialize as JSON and extract "action" value.

**Comment by lausberger** [comment]
Fixed in 640666fa17a2092cc68900e4d9b1211465b75d45

**Comment by lausberger** [comment]
Fixed in 640666fa17a2092cc68900e4d9b1211465b75d45
